beacon_node/network/src/subnet_service/tests/mod.rs

```diff
@@ -114,31 +117,20 @@ lazy_static! {
     static ref CHAIN: TestBeaconChain = TestBeaconChain::new_with_system_clock();
 }
 
-fn get_attestation_service(
-    log_level: Option<slog::Level>,
-) -> AttestationService<TestBeaconChainType> {
-    let log = get_logger(log_level);
+fn get_subnet_service() -> SubnetService<TestBeaconChainType> {
+    let log = get_logger(TEST_LOG_LEVEL);
```

Comment:
can we replace this with the `test_logger`?

Reply:
done